draft-ietf-rats-eat.md

```diff
@@ -1698,6 +1698,19 @@ an appropriate COSE hash algorithm, implying that an attacker cannot induce fals
 of a modified detached claims because the algorithms in the COSE registry are assumed to be
 of sufficient cryptographic strength.
 
+## Verification Keys {#verfication-key-sc}
+
+In all cases there must be some way that the verification key is itself verified or determined to be trustworthy.
+The key identification itself is never enough.
+This will always be by some out-of-band mechanism that is not described here.
+For example, the verifier may be configured with a root certificate or a master key by the verifier system administrator.
+
+Often an X.509 certificate or an endorsement carries more than just the verification key.
+For example, an X.509 certificate might have key usage constraints and an endorsement might have reference values.
```

Comment:
```suggestion
For example, an X.509 certificate might have key usage constraints and an Endorsement might have Reference Values.
```

Reply:
The suggestion to capitalize these terms is contrary to comments received and acted upon last summer (from Eliot I think). The change took effect in -15.

Reply:
Yes, we're using lower case.